ipinfo/handler.py

```diff
@@ -140,7 +141,10 @@ def getDetails(self, ip_address=None, timeout=None):
         response = requests.get(url, headers=headers, **req_opts)
         if response.status_code == 429:
             raise RequestQuotaExceededError()
-        response.raise_for_status()
+        if response.status_code >= 400:
```

Comment:
Is this the same status code range check as `raise_for_status()`?

Reply:
Yes, `raise_for_status()` will raise an exception if the response status code falls into the 4xx or 5xx range.